ui/component/header/view.jsx

```diff
@@ -149,15 +184,19 @@ const Header = (props: Props) => {
       <div className="header__contents">
         {!authHeader && backout ? (
           <div className="card__actions--between">
-            <Button onClick={backout.backFunction} button="link" label={__('Cancel')} icon={ICONS.ARROW_LEFT} />
-            {backout.title && (
-              <h1 className={'card__title'}>{isMobile ? backout.simpleTitle || backout.title : backout.title}</h1>
-            )}
+            <Button
+              onClick={onBackout}
+              button="link"
+              label={(backLabel && __(backLabel)) || __('Cancel')}
```

Comment:
`__(backLabel)` doesn't need to exist, because every string should already be wrapped in i18n